### PR TITLE
feat(): highlight region styles

### PR DESCRIPTION
**Pull request by jakiestfu**
BBC Team,

First off, thanks for this incredible library. Secondly, I apologize for not discussing this prior to the PR, but I hope this will be received well.

I have a strong desire to further visually customize Peaks.js. In this PR, the highlight region. Here is an example of my particular design usecase. Might be worth exploring a way to extend Konva options for every Konva object but honestly, this looks beautiful to me and I'm very happy with the ability to set these options.

Screen Shot 2022-10-12 at 8 10 04 AM

### CODE REVIEW

**Comment by jakiestfu** [comment]
@chrisn What is your interest in this? Thoughts?

**Comment by chrisn** [comment]
Very nice! Thank you for contributing. More customisation options are welcome, please raise issues with your ideas.

If you're OK with it, I may have some minor comments on the code itself.

File: src/main.js [hunk 1]
```diff
@@ -92,9 +92,12 @@ var defaultZoomviewOptions = {
 
 var defaultOverviewOptions = {
   // showPlayheadTime:    false,
-  waveformColor:       'rgba(0, 0, 0, 0.2)',
-  highlightColor:      '#aaaaaa',
-  highlightOffset:     11
+  waveformColor:          'rgba(0, 0, 0, 0.2)',
+  highlightColor:         '#aaa',
```
Comment (chrisn) [inline src/main.js:96]: I think this is causing the tests to fail

File: demo/index.html [hunk 2]
```diff
@@ -292,7 +293,7 @@ <h2>Points</h2>
           pointMarkerColor: '#006eb0',
           showPlayheadTime: true,
           waveformCache: true,
-          zoomLevels: [512, 1024, 2048, 4096]
+          zoomLevels: [512, 1024, 2048, 4096],
```
Comment (chrisn) [inline demo/index.html:296]: There's a trailing comma here

File: demo/index.html [hunk 3]
```diff
@@ -281,7 +281,8 @@ <h2>Points</h2>
             playheadClickTolerance: 3
           },
           overview: {
-            container: document.getElementById('overview-container')
+            container: document.getElementById('overview-container'),
+            // highlightColor: 'rgba(0, 0, 0, 0.1)',
```
Comment (chrisn) [inline demo/index.html:285]: Suggest removing unused code

File: src/waveform-overview.js [hunk 4]
```diff
@@ -104,7 +104,10 @@ function WaveformOverview(waveformData, container, peaks) {
   self._highlightLayer = new HighlightLayer(
     self,
     self._viewOptions.highlightOffset,
-    self._viewOptions.highlightColor
+    self._viewOptions.highlightColor,
```
[the remark this answers is not in the document]
Comment (jakiestfu) [reply src/waveform-overview.js]: Love it!

**Comment by jakiestfu** [comment]
Thanks for the code review, @chrisn! I've addressed your comments. Let me know if there is anything else I can do!

**Comment by jakiestfu** [comment]
Thanks for merging, @chrisn! When will this be available on NPM?

**Comment by chrisn** [comment]
I have just published v2.1.0. Thanks again!

**Pull request merged**